satellite/liveaccounting/liveaccounting.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package liveaccounting
```

Comment:
Wondering whether there is a better name for this package? Or maybe accounting package should have a `Live` struct instead?

Reply:
`pkg/accounting` seems like the right place by its name alone, but in code it looks to me like ithat directory is pretty tightly coupled with the accounting part of satelliteDB, and this is entirely separate from that. Secondly, I've been told that `pkg/` is for things that apply to storj code in general, not any one specific subsystem, and this code definitely only applies to a satellite. But everything seems to be in flux anyway, so I could probably be convinced that `pkg/accounting/live` or whatever is where it needs to be for now.

Reply:
Some of the packages haven’t yet been moved near their appropriate peer. That’s the reason `pkg/accounting` is still there, but there are tickets to move them.

Reply:
Some of the packages haven’t yet been moved near their appropriate peer. That’s the reason `pkg/accounting` is still there, but there are tickets to move them.

Reply:
moved to pkg/accounting/live.